src/Tasks/Microsoft.NET.Build.Tasks/ResolveAppHosts.cs

```diff
@@ -54,6 +54,8 @@ public class ResolveAppHosts : TaskBase
         public bool NuGetRestoreSupported { get; set; } = true;
 
         public string NetCoreTargetingPackRoot { get; set; }
+        
+        public bool EnableAppHostPackDownload { get; set; } = true;
```

Comment:
Normally we set the default value for something like this in an MSBuild property.  I'm not entirely sure that this works correctly if the MSBuild property is empty, though if it didn't work there would probably some test failures.